src/lib/parameters/px4params/markdownout.py

```diff
@@ -26,12 +26,20 @@ def __init__(self, groups):
             for param in group.GetParams():
                 name = param.GetName()
                 short_desc = param.GetFieldValue("short_desc") or ''
+                #short_desc = html.escape(short_desc)
```

Comment:
Why do we have `html.escape` commented-out? If we don't want it's why don't we just remove this line?